include/bits/stdio.h

```diff
@@ -2,21 +2,39 @@
 #define _BITS_STDIO_H
 
 int fprintf(FILE * stream, const char * fmt, ...) {
-	int ret;
-	va_list ap;
-	va_start(ap, fmt);
```

Comment:
These shouldn’t be defined in the header file. At a minimum they should be marked `static` if you’re gonna keep them in the header file for some reason.

Reply:
Good point. You might create an issue for this. I'm not sure we can do much about this until support for variadics lands.